### PR TITLE
[Ogbuti] - Sage/building decentralized investment platform on celo

### PR DESCRIPTION
**Pull request by dahnny**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

**Comment by developerkunal** [comment]
@dahnny your Post folder has space in name so its not taking it as post  fix that.

File: blog/02-28-23-building-a-decentralized-investment-platform copy/building-a-decentralized-investment-platform.md [hunk 1]
```diff
@@ -0,0 +1,1056 @@
...
+
+The first line is a SPDX license identifier and a Solidity version pragma.
+
+SPDX (Software Package Data Exchange) is a specification that provides a standard way to declare the license of open source software packages. The SPDX-License-Identifier is a header that specifies the license of a given source code file.
+The Solidity version pragma specifies the version of the Solidity programming language used to write the smart contract code. The version range specified is greater than or equal to 0.7.0 and less than 0.9.0, meaning that the code should be compiled with a Solidity compiler version between 0.7.0 and 0.8.x.
+
+
+Following that, we define an `IERC20Token` interface which enables us to interact with the celo stablecoin (cUSD). 
+
+```js
```
Comment (developerkunal) [inline blog/02-28-23-building-a-decentralized-investment-platform copy/building-a-decentralized-investment-platform.md:65]: > make it ```solidity

File: blog/02-28-23-building-a-decentralized-investment-platform copy/building-a-decentralized-investment-platform.md [hunk 2]
```diff
@@ -0,0 +1,1056 @@
...
+
+ERC-20 tokens are a widely-used standard for creating digital assets on the Ethereum blockchain, and cUSD is one of them.
+
+These tokens have pre-defined functions and events that can be easily used in contracts and do not require any additional implementation. For example, you will be using the ERC-20 token's interface to interact with it, so that your contract can communicate with the token.
+
+You can find more information on how to use these functions and events in the Celo **[documentation](https://docs.celo.org/developer-guide/celo-for-eth-devs)**. The documentation also provides more details on how to interact with ERC-20 tokens and how to use them with the Celo network.
+
+Next up, you would need to create a contract(in this case you name the contract `celoinvest`) and then define a way for investments to be tracked on the blockchain, including who made the investment, how much they invested, and when the investment matures and is paid back.
+
+```js
```
Comment (developerkunal) [inline blog/02-28-23-building-a-decentralized-investment-platform copy/building-a-decentralized-investment-platform.md:91]: > make it ```solidity for all solidity code

File: blog/02-28-23-building-a-decentralized-investment-platform copy/building-a-decentralized-investment-platform.md [hunk 3]
```diff
@@ -0,0 +1,1056 @@
...
+    </section>
+  );
+};
+
+export default MyInvestments;
+```
+
+After completing the development of your React DApp, you should attempt to compile it to ensure that it is functioning correctly. If everything appears to be working correctly, you can then deploy your DApp on platforms such as Github Pages or Netlify. For reference, you can use the project available at <https://github.com/dahnny/celoinvest> as an example and obtain necessary files and images to edit your own project.
+
+## Concl
```
Comment (developerkunal) [inline blog/02-28-23-building-a-decentralized-investment-platform copy/building-a-decentralized-investment-platform.md:1047]: i guess conclusion is missing over here

**Comment by developerkunal** [comment]
There is some changes done with src/data-tutorials/showcase/intermediate/how-to-build-car-marketplace-using-react.png this image don't do it. its someone else file

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**